### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by eamador1**
Since the last code review, the following was done in this project:
---

- Add any missing JSX elements to project (without adding extra functionality) ✅
- final design match Zeplin's design (this includes all elements + the same styling) ✅
- Use the CSS rules detailed in Zeplin to style the website ✅

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,14 +7,45 @@ import {
...
+            activeClassName="active-link"
+            className={`CATEGORIES ${isCategoriesActive ? 'active-link' : ''}`}
+            onClick={handleCategoriesClick}
+          >
+            Categories
+          </NavLink>
+          <div className="oval">
+            <SvgUser className="Mask" />
+          </div>
         </nav>
```
Comment (shahadat3669) [inline src/App.js]: - [ ] Your navbar component is a great addition to your app; however, it currently doesn't align with the provided design. To enhance your app's aesthetics and user experience, please ensure that you use the specified layout to match the design. Please ensure that the navbar is full with the book's card. By incorporating these design elements, you'll bring your navbar closer to the intended design, improving the overall look and feel of your app. Keep up the good work! 👍🎨

| Your | Design |
| --- | --- |
|image|  image|

File: src/components/Form.jsx [hunk 2]
```diff
@@ -25,18 +25,20 @@ function Form() {
...
       <input
+        className="bookAuthor"
         type="text"
         placeholder="Author"
         value={author}
         onChange={(e) => setAuthor(e.target.value)}
       />
-      <button type="submit">Add Book</button>
+      <button className="submitButton" type="submit">Add Book</button>
     </form>
```
Comment (shahadat3669) [inline src/components/Form.jsx]: - [ ] 👍 Good work so far! However, there's a minor layout issue. As per the design, your input fields should take up the full width until the button, and the button should be aligned horizontally with the card on the right side. Currently, they are not aligned as intended. Please update your code to ensure that the layout matches the design, providing a more polished and visually consistent appearance. Keep it up! 🛠️👏🎨

| Your | Design |
| --- | --- |
|  image| image|

File: src/components/Book.jsx [hunk 3]
```diff
@@ -11,17 +11,53 @@ const Book = ({ item }) => {
...
+          <div className="circle-wrap">
+            <div className="circle">
+              <div className="mask half">
+                <div className="fill">
+                  <div className="mask full">
+                    <div className="inside-circle" />
+                  </div>
+                </div>
+              </div>
+            </div>
```
Comment (shahadat3669) [inline src/components/Book.jsx]: - [ ] It looks good that you added a circular progress bar, but it is smaller compared to the design. Please consider updating it.

| Your | Design |
| --- | --- |
|  image| image|

File: src/components/Book.jsx [hunk 4]
```diff
@@ -11,17 +11,53 @@ const Book = ({ item }) => {
...
+          {' '}
+          {item.author}
+        </span>
+        <div className="allButtons">
+          <button className="CommentsButton" type="button">Comments</button>
+          <button className="RemoveButton" type="button" onClick={handleDelete}>
+            Remove
+          </button>
+          <button className="EditButton" type="button">Edit</button>
+        </div>
```
Comment (shahadat3669) [inline src/components/Book.jsx:27]: 👍 Good job with the layout! However, there are a couple of improvements needed to match the design precisely:

- [ ] Firstly, as per the design, there should be no horizontal line after the last button, which is the 'Edit' button. Please update your code to remove the line after the 'Edit' button, aligning it with the design.

- [ ] Secondly, the horizontal lines should be evenly spaced between the buttons. Currently, they are not evenly spaced. Please adjust your code to ensure that the horizontal lines are evenly distributed between each button, adhering to the design specifications.

| Your | Design |
| --- | --- |
|  image| image|

These adjustments will help your layout closely match the design, enhancing its visual consistency. Great work, and keep refining the details! 🎨👍✨

File: src/App.js
Comment (shahadat3669) [inline src/App.js:1]: 👍 Great work on the components! However, to align them more closely with the design, consider the following improvements:

- [ ] Wrap all components within an outer wrapper element. This wrapper should have a shadow effect to match the design's visual style. Incorporating a shadow will provide a more polished and professional appearance. Pay attention to the spacing and layout, ensuring that elements are properly aligned and spaced as per the design specifications. You can refer to this [example](https://pedantic-beaver-cb923b.netlify.app/#/) to understand the desired layout.


| Your | Design |
| --- | --- |
|  image| Bookstore CMS (1)|


By adding the outer wrapper with a shadow and fine-tuning the spacing, your components will closely match the design, creating a more visually appealing and consistent user interface. Keep up the excellent work! 🎨👍✨

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @eamador1, 

Great job so far! 👏 You've done well, and you're almost there for the next project ⏭️, but there are a few issues 🐛 that you still need to work on.




## Don't Worry 🙏

It's okay to have bugs in your code. 🆗 Please see this code review as an opportunity to learn practical coding skills and discover new tips that can improve your code quality 💯 and help you **get even better** 💼.

**I will guide you on how to address these areas down below 👇.**

## Highlights
✅ No linter Errors
✅ No repeated id or class
✅ The code is top-notch! 💯


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eamador1** [comment]
Since the last code review, the following changes were made:
---
-wrap all components
-fix vertical lines in button
-circular bar has same width and height as example
-align all components

File: src/App.css [hunk 5]
```diff
@@ -15,13 +14,86 @@ a {
   color: inherit;
 }
```
Comment (mukhammadrizooff) [inline src/App.css:16]: - [ ] [REQUIRED] Good job so far! As previous reviewer mentioned about your main box of project should be give `shadow` property and pay attention to `distance of elements from the border of box, in your project I see these aspects are not fixed yet. Please improve them 🙏🏼 

image
  
image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @eamador1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eamador1** [comment]
Since the last code review, the following was done:
---

- import react-circular-progressbar library ✅
- give shadow property to books container ✅
- give proper distance to elements in app ✅

**Review by ShadyShawkat — APPROVED** [review]
## Hi @eamador1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**